packages/utils/src/errors.ts

```diff
@@ -0,0 +1,149 @@
+/**
+ * Types
+ */
+type SdkErrorKey = keyof typeof SDK_ERRORS;
+type InternalErrorKey = keyof typeof INTERNAL_ERRORS;
+
+/**
+ * Constants
+ */
+const SDK_ERRORS = {
+  INVALID_METHOD: {
```

Comment:
Style (optional):

Could be nice to label these by error class, similar to how they are in the spec, e.g.:

```js
  /* ----- INVALID (1xxx) ----- */
  INVALID_METHOD: {
  // ...
  /* ----- UNAUTHORIZED (3xxx) ----- */
  UNAUTHORIZED_METHOD: {
```